### PR TITLE
update migration doc and reference migration in main readme

### PR DESCRIPTION
**Pull request by brianashby-sfx**

### CODE REVIEW

File: migration.md [hunk 1]
```diff
@@ -1,118 +1,124 @@
...
+# Migrate from the SignalFx Tracing Library for Python
+
+The Splunk Distribution of OpenTelemetry Python replaces the SignalFx Tracing
+Library for Python. If you’re using the SignalFx Tracing Library, migrate to
+the Splunk Distribution of OpenTelemetry Python to use OpenTelemetry’s
+instrumentation to send traces to Splunk APM. The Splunk Distribution of
+OpenTelemetry Python uses OpenTelemetry to instrument applications, which is
+an open-source API to gather telemetry data, and has a smaller footprint.
+
+Because the SignalFx Java Agent uses OpenTracing and the Splunk Distribution
```
Comment (owais) [inline migration.md:10]: Java ?

File: migration.md [hunk 2]
```diff
@@ -1,118 +1,124 @@
...
+of OpenTelemetry Java Instrumentation uses OpenTelemetry, the semantic
+conventions for span tag names change when you migrate. For more information,
+see [Migrate from OpenTracing to OpenTelemetry](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html#apm-opentelemetry-migration).
+
+This Splunk Distribution of OpenTelemetry can't send traces directly to a
```
Comment (owais) [inline migration.md:15]: It can. Main readme shows how

File: migration.md [hunk 3]
```diff
@@ -1,118 +1,124 @@
...
+
+Migrate settings from the SignalFx tracing library to the Splunk Distribution
+for OpenTelemetry:
+
+Rename required environment variables:
+    | Old Environment Variable           | New Environment Variable             |
+    | ---------------------------------- | ------------------------------------ |
+    | SIGNALFX_SERVICE_NAME              | SPLK_SERVICE_NAME                    |
+    | SIGNALFX_ENDPOINT_URL              | OTEL_EXPORTER_ZIPKIN_ENDPOINT        |
+    | SIGNALFX_RECORDED_VALUE_MAX_LENGTH | SPLK_MAX_ATTR_LENGTH                 |
```
Comment (owais) [inline migration.md:112]: Should add SPLK_ACCESS_TOKEN to the table

File: migration.md [hunk 4]
```diff
@@ -1,118 +1,124 @@
...
+
+### Step 4. Update your Django configuration
+
+This Splunk Distribution of OpenTelemetry doesn't ship a Django app, and
+doesn't require you to add anything to your `INSTALLED_APPS`.
+
+Follow these settings to update your Django configuration:
```
Comment (owais) [inline migration.md:119]: s/settings/steps

File: migration.md [hunk 5]
```diff
@@ -1,118 +1,124 @@
...
-
-splunk-opentelemtry no longer ships a Django app and does not require you to add anything to your `INSTALLED_APPS` or any other
-setting from settings.py
-
-#### Set `DJANGO_SETTINGS_MODULE` environment variable
-
-splunk-opentelemetry requires `DJANGO_SETTINGS_MODULE` environment to be set before it can auto-instrument a Django project. This
-should be the same as the value set in your manage.py or wsgi.py app and should be a valid Python import path.
-
-## Known issues
```
Comment (owais) [inline migration.md:111]: This section had been removed. Should be added bank

Comment (brianashby-sfx) [reply migration.md:111]: - OpenTelemetry's psycopg2 has a bug that prevents the driver from functioning correctly. Thus it has been disabled temporarily in this package.
-- this isn't specific to the migration.
- Does not yet support sending spans directly to the SignalFx backend and requires the SignalFx Smart Agent or OpenTelemetry collector.
-- this is no longer true.
- Does not yet support enabling or disabled specific instrumentations.
-- this is not a known issue of the migration or the package.
- Does not yet support tracing and log correlation.
-- this is not a known issue of the migration of the package.

Comment (owais) [reply migration.md:111]: Other than the second point, all of them are known issues and relate directly to migration. These are features that work with signalfx-python-tracing but do not work with splunk-otel-python so migrating will cause these regressions and people should know about this. It's very important to call these out so users aren't surprised after completing the migration only to find out these things not working.

File: migration.md [hunk 6]
```diff
@@ -1,118 +1,128 @@
...
-- OpenTelemetry's psycopg2 has a bug that prevents the driver from functioning correctly. Thus it has been disabled temporarily in this package.
-- Does not yet support sending spans directly to the SignalFx backend and requires the SignalFx Smart Agent or OpenTelemetry collector.
-- Does not yet support enabling or disabled specific instrumentations.
-- Does not yet support tracing and log correlation. 
\ No newline at end of file
+These are the known issues as a result of migrating from the SignalFx Tracing Library for Python to the Splunk Distribution of OpenTelemetry Python:
+
+- psycopg2 is disabled because it contains a bug that prevents the driver from functioning correctly.
+- You can't enable or disable specific instrumentations.
+- You can't inject trace context in logs.
```
Comment (owais) [inline migration.md:21]: We should clarify that these features be added soon.

File: migration.md [hunk 7]
```diff
@@ -1,118 +1,128 @@
...
+
+Migrate settings from the SignalFx tracing library to the Splunk Distribution
+for OpenTelemetry:
+
+Rename required environment variables:
+    | Old Environment Variable           | New Environment Variable             |
+    | ---------------------------------- | ------------------------------------ |
+    | SIGNALFX_ACCESS_TOKEN              | SPLK_ACCESS_TOKEN
+    | SIGNALFX_SERVICE_NAME              | SPLK_SERVICE_NAME                    |
+    | SIGNALFX_ENDPOINT_URL              | OTEL_EXPORTER_ZIPKIN_ENDPOINT        |
```
Comment (owais) [inline migration.md:115]: This should be `SPLK_TRACE_EXPORTER_URL`

File: migration.md (hunk 6 above)
Comment (brianashby-sfx) [reply migration.md]: i updated the known issues about features to say you can't do something "right now," but don't want to explicitly say they will be added soon in the migration doc.

**Review by owais — APPROVED** [review]

**Pull request merged**